pages/api/capture-event.js

```diff
@@ -11,12 +11,30 @@ export default withContainer(
       return;
     }
 
-    const userIsAuthenticated = container.getUserIsAuthenticated();
-    const userIsAuthenticatedResponse = await userIsAuthenticated(
-      headers.cookie
-    );
+    if (headers?.cookie) {
```

Comment:
This makes the assumption that if they have a cookie set, then they should go through the normal authentication. If we were to store a cookie on key contact visitors, this may stop capturing their events in the future. 

What if we skip the `headers?.cookie` check and do both the user authentication and the call password verification, returning unauthorised if both return false?